### PR TITLE
doc: add emoji for attendees

### PR DESCRIPTION
**Pull request by marco-ippolito**
@nodejs/security-wg

### CODE REVIEW

File: templates/meeting_base_security-wg [hunk 1]
```diff
@@ -8,3 +8,15 @@ JOINING_INSTRUCTIONS="https://zoom.us/j/92309450775
 * link for participants: <>
 * For those who just want to watch We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at <https://www.youtube.com/c/nodejs+foundation/live> when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.
 * youtube admin page: <https://www.youtube.com/my_live_events?filter=scheduled>"
+
```
[the remark this answers is not in the document]
Comment (marco-ippolito) [reply templates/meeting_base_security-wg:11]: Strangely I found that quote other in all other templates, I dont know why 🤔

[the remark this answers is not in the document]
Comment (mhdawson) [reply templates/meeting_base_security-wg:11]: @marco-ippolito thanks will take another look.

[the remark this answers is not in the document]
Comment (mhdawson) [reply templates/meeting_base_security-wg:11]: @marco-ippolito I looked at a few and don't see it in the TSC one. I just see one right after JOINING_INSTRUCTIONS= and then one at the end?

**Review by fraxken — APPROVED** [review]

File: templates/meeting_base_security-wg (hunk 1 above)
[the remark this answers is not in the document]
Comment (marco-ippolito) [reply templates/meeting_base_security-wg:11]: yes I removed the double quote in the middle, should be equal to https://github.com/nodejs/create-node-meeting-artifacts/blob/main/templates/meeting_base_tsc

**Review by UlisesGascon — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**